lib/view/pages/home_page.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:superformula_test/core/resources/extensions.dart';
+import 'package:superformula_test/view/widgets/extended_menu_fab.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  double _imageScale = 0;
+  double _textXOffset = -3;
+
+  @override
+  void initState() {
+    super.initState();
+    setAnimationTimer();
```

Comment:
**question**: why didn't you use an `AnimationController` here? How would you (for example) run this animation backwards or pause/resume it with a `Timer`?

Check the [documentation](https://api.flutter.dev/flutter/animation/AnimationController-class.html) to see how an `AnimationController` works 🙂

Reply:
With AnimationController I will probably need to use explicit animations. Since at this page I am using Implicit animations and the animations will be triggered just one time (when the screen is opened), I used a Timer just to "After 100 ms when the screen showed up, please update these values (then my implicit animation would be triggered)".

But using setState to update all the screen just to start the animations is not a good approach as you mentioned. Thanks for letting me know!